sky/build/sky_precompilation_sdk.gni

```diff
@@ -92,4 +94,30 @@ template("sky_precompilation_sdk") {
       ":$copy_config_gen_target_name",
     ]
   }
+
+  sdk_zip_gen_target_name = target_name + "_zip"
+  action(sdk_zip_gen_target_name) {
+    zip_path = "$root_out_dir/$sdk_name.zip"
+    script = "//sky/build/zip_xcodeproj.py"
+    depfile = "$target_gen_dir/$sdk_zip_gen_target_name.d"
+    outputs = [
+      depfile,
+      zip_path,
+    ]
+    args = [
+      "--depfile",
+      rebase_path(depfile, root_build_dir),
+      "--input",
+      rebase_path(sdk_dir),
```

Comment:
At some point, we should switch over from using the build system to package up the SDK and into a script that runs on the buildbot and uploads the result to google storage.  Maybe zipping up the build products is the place to switch over?  Then you don't have any dependency-tracking problems because you always run that process from scratch (i.e., no incremental builds).